.github/workflows/update_website.yaml

```diff
@@ -0,0 +1,19 @@
+name: Trigger website update
+on:
+  push:
+    tags: "v*"
+jobs:
+  dispatch:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/github-script@v6
+      with:
+        github-token: "${{ secrets.PAT_TOKEN }}"
```

Comment:
Can we rename this to `ACCESS_TOKEN` or `PERSONAL_ACCESS_TOKEN` to make it consistent with the name I chose to deploy the website ?

![image](https://user-images.githubusercontent.com/8488398/164086580-315af3cd-3082-4cd6-823b-a14d9da1bcdc.png)

Is this _in addition to_ the GitHub actions on the jmespath.site repo or is this completely replacing the logic to deploy?

Reply:
sure thing, that name was just copied from some tutorial. Now that I think of it the "T" in PAT is redundant.

The way this works is this workflow does nothing more than make an api request on the site repo to run its build workflow.
The site repo needs to be finalised before we merge this.

Reply:
Actually, we can merge this but shouldn't tag the repo until everything is in place.